### PR TITLE
test(cypress): add FRM Routing Algorithm test coverage (QAC-115)

### LINKED ISSUE #12011: [QAC-115] Add FRM Routing Algorithm test coverage
## Objective
Add Cypress test coverage for FRM (Fraud Risk Management) Routing Algorithm field in merchant account creation and management APIs.

## Scope
- Test FRM routing algorithm field during merchant account creation
- Test priority and single routing algorithm types
- Verify persistence on retrieve/update operations
- Test edge cases for invalid algorithm structures

## Acceptance Criteria
- [ ] Create merchant account with single FRM routing algorithm
- [ ] Create merchant account with priority FRM routing algorithm  
- [ ] Retrieve merchant account and verify FRM routing algorithm persists
- [ ] Update merchant account with FRM routing algorithm
- [ ] Create merchant account without FRM routing algorithm
- [ ] Test edge cases (missing type, missing data, empty object, unknown type)

## Related PR
#12009

## Test Results
- Stripe regression: PASS (12/12 tests)
- FRM spec tests: PASS (9/9 tests)
- Account create prereqs: PASS (3/3 tests)

## Labels
QA, Cypress, Test Coverage

### PR DESCRIPTION
**Pull request by venkatakarthikm-maker**
## Type of Change
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [x] QA/Test

## Description
Closes #12011

## What changed
- Added Cypress test coverage for FRM Routing Algorithm field in merchant account creation
- Created `cypress/e2e/spec/FRM/00000-FrmRoutingMerchantAccount.cy.js` with 8 test cases
- Created `cypress/fixtures/frm-routing-test-data.json` with test data
- Updated `cypress/fixtures/imports.js` to include new fixture

## Test Cases Added
1. Create merchant with single frm_routing_algorithm
2. Create merchant with priority frm_routing_algorithm
3. Retrieve merchant and verify persistence
4. Update merchant with frm_routing_algorithm
5. Create merchant without frm_routing_algorithm
6. Edge case: missing type field
7. Edge case: missing data field
8. Edge case: empty object

## How did you test it?
- Stripe regression: PASS (12/12 tests)
- FRM spec tests: PASS (9/9 tests)
- Account create prereqs: PASS (3/3 tests)

## GATE
GATE_PASSED:
  StripeRegression: PASS
  AllChecksPassed: YES
  ReadyForGitHubAgent: YES

### CODE REVIEW

File: cypress-tests/cypress/e2e/spec/FRM/00000-FrmRoutingMerchantAccount.cy.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+      const createBody = JSON.parse(
+        JSON.stringify(fixtures.merchantCreateBody)
+      );
+      createBody.merchant_id = merchantId;
+      createBody.frm_routing_algorithm =
+        fixtures.frmRoutingTestData.valid_single_routing;
+
+      globalState.set("frmMerchantId", merchantId);
+
+      cy.request({
```
Comment (likhinbopanna) [inline cypress-tests/cypress/e2e/spec/FRM/00000-FrmRoutingMerchantAccount.cy.js]: Can you use commands.js for all the api calls and assertions ?
